cds_ils/utils.py

```diff
@@ -16,3 +16,32 @@ def format_login_required_urls(urls):
             url["login_required_url"] = current_app.config[
                 "CDS_ILS_EZPROXY_URL"
             ].format(url=url["value"])
+
+
+def dynamic_delete_field(original_element, fields):
+    """Dynamically delete an array of fields from a dictionary."""
+    # Wrong parameter in the configuration
+    wrong_field = False
+
+    for field in fields:
+        if not isinstance(field, list):
+            original_element.pop(field)
+            continue
+        if not field:
+            continue
+
+        element = original_element
+        for subfield in field[:-1]:
+            if subfield in element.keys():
+                element = element[subfield]
+            else:
+                wrong_field = True
+                break
```

Comment:
Wrong field is used when you try to access of sub-field of an incorrect field, to skip the delete. Ej: if the configuration is [links [metodoto, pid]], we wouldn't get an error when why try to delete access pid in the non existing metodoto.

Reply:
I do not fully understand the concept, I think we should discuss in person